experimental_official_dockerfiles/master/setup.devel.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+ 
+################################################################################
+
+set -e
+
+# Retry on connection timeout.
+bash -c "echo 'APT::Acquire::Retries \"3\";' > /etc/apt/apt.conf.d/80-retries"
+
+# Install bootstrap dependencies from ubuntu deb repository.
+apt-get update
+apt-get install -y --no-install-recommends \
+    apt-transport-https ca-certificates software-properties-common
+apt-get clean
+rm -rf /var/lib/apt/lists/*
+
+################################################################################
+
+set -e
+ubuntu_version=$(cat /etc/issue | grep -i ubuntu | awk '{print $2}' | \
+  awk -F'.' '{print $1}')
+
+if [[ "$1" != "" ]] && [[ "$1" != "--without_cmake" ]]; then
+  echo "Unknown argument '$1'"
+  exit 1
+fi
+
+# Install dependencies from ubuntu deb repository.
+apt-key adv --keyserver keyserver.ubuntu.com --recv 084ECFC5828AB726
+apt-get update
+
+if [[ "$ubuntu_version" == "14" ]]; then
+  # specifically for trusty linked from ffmpeg.org
+  add-apt-repository -y ppa:mc3man/trusty-media
+  apt-get update
+  apt-get dist-upgrade -y
+fi
+
+## TODO(yifeif) remove ffmpeg once ffmpeg is removed from contrib
```

Comment:
We don't have contrib anymore

Reply:
Thanks for the reminder. I don't actually use this file, it's just here for reference.